source/add-jupyter/customize-attributes.rst

```diff
@@ -0,0 +1,129 @@
+.. _add-jupyter-customize-attributes:
+
+Customize Attributes
+====================
+
+Now we will customize the app to work on a given cluster. Be sure that you
+walked through :ref:`add-jupyter-software-requirements` for the given cluster
+ahead of time.
+
+Modify Form File
+----------------
+
+The main responsibility of the ``form.yml`` file is for defining the attributes
+(their values or HTML form elements) used when generating the batch script.
+
+#. We will begin by adding a cluster for the Jupyter app to use. You do this by
+   editing the ``form.yml`` in your favorite editor as such:
+
+   .. code-block:: yaml
+
+      # form.yml
+      ---
+
+      ...
+
+      cluster: "my_cluster"
+
+      ...
+
+   where we replace ``my_cluster`` with a valid cluster that corresponds to a
+   cluster configuration file located under
+   :file:`/etc/ood/config/clusters.d/{cluster}.yml`.
+
+#. We will also edit the module(s) that is(are) required to be loaded within
+   our batch job to get a Jupyter Notebook Server running:
+
+   .. code-block:: yaml
+
+      # form.yml
+      ---
+
+      ...
+
+      attributes:
+
+        ...
+
+        modules: "python/3.5"
+
+      ...
+
+   where we replace ``python/3.5`` with a list of required modules for our
+   given cluster.
+
+#. **Optional** If we do NOT have the Conda extensions installed for the above
+   Python modules then we must disable it:
+
+   .. code-block:: yaml
+
+      # form.yml
+      ---
+
+      ...
+
+      attributes:
+
+        ...
+
+        conda_extensions: "0"
+
+      ...
+
+   They are enabled by default in this file.
+
+.. note::
+
+   It is recommended you commit the changes you made to ``form.yml`` to `git`_:
+
+   .. code-block:: sh
+
+      # Stage the modified file
+      git add form.yml
```

Comment:
See note above about mentioning where this file is located. If it's preexisting and needs to be edited, I think having a `git add` command here makes it feel like we're creating, rather than editing this file.

If it already exists, maybe just use `git commit form.yml -m 'updated form with cluster attributes'`